cockatrice/src/dialogs/dlg_settings.cpp

```diff
@@ -588,8 +588,9 @@ DeckEditorSettingsPage::DeckEditorSettingsPage()
     connect(urlList->model(), SIGNAL(rowsMoved(const QModelIndex, int, int, const QModelIndex, int)), this,
             SLOT(urlListChanged(const QModelIndex, int, int, const QModelIndex, int)));
 
-    for (int i = 0; i < SettingsCache::instance().downloads().getCount(); i++)
-        urlList->addItem(SettingsCache::instance().downloads().getDownloadUrlAt(i));
+    for (QString url : SettingsCache::instance().downloads().getAllURLs()) {
+        urlList->addItem(url);
```

Comment:
why do we use a loop here while at line 700 we use addItems to add all in a single line?

Reply:
good point